code/modules/hydroponics/grown/flowers.dm

```diff
@@ -15,7 +15,7 @@
 	icon_grow = "poppy-grow"
 	icon_dead = "poppy-dead"
 	mutatelist = list(/obj/item/seeds/poppy/geranium, /obj/item/seeds/poppy/lily)
-	reagents_add = list("bicaridine" = 0.2, "nutriment" = 0.05)
+	reagents_add = list("bicaridine" = 0.2, "plantmatter" = 0.05)
```

Comment:
Because those are horrifically toxic, while ingested; these are meant to provide medicinal benefit when consumed or utilized---not poison you horrendously when you eat a "healthy" salad.